TFT/src/User/API/Vfs/vfs.c

```diff
@@ -132,63 +132,88 @@ bool IsRootDir(void)
 }
 
 // check if filename provides a supported filename extension
-char * isSupportedFile(char * filename)
+char *isSupportedFile(char * filename)
 {
-  char * extPos = strstr(filename, ".g");  // support "*.g","*.gco" and "*.gcode"
+  char *extPos = strrchr(filename, '.');  // check last "." in the name where extension is supposed to start
 
-  if (extPos == NULL)
-    extPos = strstr(filename, ".G");  // support "*.g","*.gco" and "*.gcode"
+  if (extPos != NULL && extPos[1] != 'g' && extPos[1] != 'G')
+  {
```

Comment:
I would prefer to avoid the use of {} with a single line. but this is only my preferred style

Reply:
You can avoid it, no problem.